tests/integration/components/title-user-test.js

```diff
@@ -1,13 +0,0 @@
-import { module, test } from 'qunit';
-import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
-import hbs from 'htmlbars-inline-precompile';
-
-module('Integration | Component | title user', function(hooks) {
-  setupRenderingTest(hooks);
-
-  test('it renders without causing failures down the line', async function(assert) {
-    await render(hbs`{{title-user}}`);
```

Comment:
This was originally added about 4 years go - https://github.com/adopted-ember-addons/ember-page-title/commit/86a30ade6104ca1f989c2421615b9c409c721915

Reply:
👍